packages/mattermost-redux/src/reducers/entities/search.ts

```diff
@@ -71,7 +71,7 @@ function matches(state: Record<string, string[]> = {}, action: GenericAction) {
     }
     case SearchTypes.REMOVE_SEARCH_POSTS:
     case UserTypes.LOGOUT_SUCCESS:
-        return [];
```

Comment:
A few of these changes are actually fixing some places where we changed the type of the state accidentally